app/src/main/java/org/jellyfin/androidtv/ui/playback/nextup/NextUpButtons.kt

```diff
@@ -9,21 +9,21 @@ import android.widget.FrameLayout
 import android.widget.ProgressBar
 import androidx.appcompat.widget.AppCompatButton
 import org.jellyfin.androidtv.R
-import org.jellyfin.androidtv.preference.UserPreferences
-import org.koin.core.component.KoinComponent
-import org.koin.core.component.get
 
 class NextUpButtons(
 	context: Context,
 	attrs: AttributeSet? = null,
 	defStyleAttr: Int = 0,
 	defStyle: Int = 0
-) : FrameLayout(context, attrs, defStyleAttr, defStyle), KoinComponent {
+) : FrameLayout(context, attrs, defStyleAttr, defStyle) {
 	constructor(context: Context, attrs: AttributeSet) : this(context, attrs, 0, 0)
 
 	private var countdownTimer: CountDownTimer? = null
 	private val view = View.inflate(context, R.layout.fragment_next_up_buttons, null)
 
+	var countdownTimerEnabled: Boolean = true
+	var duration: Long = 6
```

Comment:
```suggestion
	var duration: Int = 6
```

Also use an int as the preference is an int too. Integers are big enough for this usecase. Alternatively a Kotlin Duration instance could be used but that's probably overkill.